models/Item.js

```diff
@@ -0,0 +1,58 @@
+const mongoose = require("mongoose");
+
+const itemSchema = new mongoose.Schema({
+    // var name below TBC
+    postID: {
+        type: Number,
+        required: true,
+    },
```

Comment:
IDs will be automatically generated when a document is created so postID is not necessary